web/src/App.js

```diff
@@ -34,11 +34,11 @@ function App() {
           <Route path="/" element={<Main />} />
           <Route path="/professor/:professorId" element={<ProfessorPage />} />
         </Routes>
+        <Comment />
```

Comment:
I think the way this is set up it is going to render Comment on every page now, similar to the Header and Footer since they're outside of the routes. I think it might need to be called within the ProfessorPage.js instead of in App.js like how Chart is called within ProfessorPage.js, that way it is just loaded within that page